demos/python_demos/gesture_recognition_demo/README.md

```diff
@@ -77,7 +77,7 @@ python gesture_recognition_demo.py \
 -m_a /home/user/asl-recognition-0004.xml \
 -m_d /home/user/person-detection-asl-0001.xml \
 -i 0 \
--c ./msasl100-classes.json
+-c <omz_dir>/data/datasets_classes/msasl100-classes.json
```

Comment:
There are other links below, though they point to nonexistent `./classes.json`. Can you update them please.

Reply:
done